enrolments/tests/serializers/test_session_create.py

```diff
@@ -43,7 +43,6 @@ def setUp(self):
 
         self.name = "Fall 2020"
         self.start_date = date(2020, 9, 5)
-        self.fields = [1, 2, 3]
```

Comment:
was this variable not being used?

Reply:
Noo, I'm using `field_ids`
Just forgot to remove this variable in my last PR that went in